src/test/regress/expected/isolation_dump_global_wait_edges.out

```diff
@@ -0,0 +1,85 @@
+Parsed test spec with 4 sessions
+
+starting permutation: s1-begin s2-begin s1-update s2-update detector-dump-wait-edges s1-abort s2-abort
+step s1-begin: 
+    BEGIN;
+
+step s2-begin: 
+    BEGIN;
+
+step s1-update: 
+    UPDATE distributed_table SET y = 1 WHERE x = 1;
+
+step s2-update: 
+    UPDATE distributed_table SET y = 2 WHERE x = 1;
+ <waiting ...>
+step detector-dump-wait-edges: 
+    SELECT
+        waiting_transaction_num,
+        blocking_transaction_num,
+        blocking_transaction_waiting
+    FROM
+        dump_global_wait_edges()
+    ORDER BY
+        waiting_transaction_num,
+        blocking_transaction_num,
+        blocking_transaction_waiting;
+
+waiting_transaction_numblocking_transaction_numblocking_transaction_waiting
+
+189            188            f              
+step s1-abort: 
+    ABORT;
+
+step s2-update: <... completed>
+step s2-abort: 
+    ABORT;
+
+
+starting permutation: s1-begin s2-begin s3-begin s1-update s2-update s3-update detector-dump-wait-edges s1-abort s2-abort s3-abort
+step s1-begin: 
+    BEGIN;
+
+step s2-begin: 
+    BEGIN;
+
+step s3-begin: 
+    BEGIN;
+
+step s1-update: 
+    UPDATE distributed_table SET y = 1 WHERE x = 1;
+
+step s2-update: 
+    UPDATE distributed_table SET y = 2 WHERE x = 1;
+ <waiting ...>
+step s3-update: 
+    UPDATE distributed_table SET y = 3 WHERE x = 1;
+ <waiting ...>
+step detector-dump-wait-edges: 
+    SELECT
+        waiting_transaction_num,
+        blocking_transaction_num,
+        blocking_transaction_waiting
+    FROM
+        dump_global_wait_edges()
+    ORDER BY
+        waiting_transaction_num,
+        blocking_transaction_num,
+        blocking_transaction_waiting;
+
+waiting_transaction_numblocking_transaction_numblocking_transaction_waiting
+
+193            192            f              
```

Comment:
Yeh, I thought about coming up with some CTE that maps values consistently, but it gets pretty complicated.